### PR TITLE
Google Login

### PR DESCRIPTION
**Pull request by DerScreever**
This pull request includes a

- [ ] Bug fix
- [x] New feature
- [ ] Translation

The following changes were made

- Added google login

If this is related to an existing ticket, include a link to it as well.

### CODE REVIEW

File: example.py [hunk 1]
```diff
@@ -365,8 +380,9 @@ def main():
     pokemonsJSON = json.load(open(path + '/pokemon.json'))
 
     parser = argparse.ArgumentParser()
-    parser.add_argument("-u", "--username", help="PTC Username", required=True)
-    parser.add_argument("-p", "--password", help="PTC Password", required=True)
+    parser.add_argument("-a", "--auth_service", help="Auth Service",required=True)
```
Comment (nborrmann) [inline example.py]: Add the possible values to the help string. Possibly make it default to 'ptc'.

**Comment by sykhro** [comment]
Please rebase this

**Comment by AHAAAAAAA** [comment]
+1

**Comment by DerScreever** [comment]
fixed the conflicts

**Pull request merged**